src/commands/dev/dev.ts

```diff
@@ -155,6 +157,16 @@ export const dev = async (options: OptionValues, command: BaseCommand) => {
     siteInfo,
   })
 
+  if (!options.offline && !options.offlineEnv) {
+    await setupAIGateway({ api, env, siteID: site.id, siteURL: siteUrl })
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (env.AI_GATEWAY?.value) {
+    process.env.AI_GATEWAY = env.AI_GATEWAY.value
```

Comment:
Injecting the environment variables into the process should happen automatically in the `injectEnvVariables` call above (L148). Any chance we could move the AI gateway initialisation above it so that we don't need to do this manually? I think we'd need to move the `getSiteInformation` call up a bit, but that should be okay.